de.bforartists.Bforartists.json

```diff
@@ -0,0 +1,179 @@
+                "--enable-decoder=aac",
+                "--enable-decoder=h264",
+                "--enable-decoder=libvorbis",
+                "--enable-decoder=libvpx_vp8",
+                "--enable-decoder=libvpx_vp9",
+                "--enable-decoder=mp3",
+                "--enable-decoder=mpeg4",
+                "--enable-decoder=pcm_s16le",
+                "--enable-decoder=theora"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.ffmpeg.org/releases/ffmpeg-5.0.1.tar.xz",
+                    "sha256": "ef2efae259ce80a240de48ec85ecb062cecca26e4352ffb3fda562c21a93007b"
+                }
+            ],
+            "cleanup": [
+                "/lib/codecs/include",
+                "/lib/codecs/lib/pkgconfig",
+                "/lib/codecs/share"
+            ]
+        },
+        {
+            "name": "bforartists",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm755 bforartists.sh /app/bin/bforartists",
+                "mv bforartists /app/bforartists",
+                "install -Dm644 /app/bforartists/bforartists.desktop /app/share/applications/$FLATPAK_ID.desktop",
+                "desktop-file-edit --set-icon=${FLATPAK_ID} /app/share/applications/${FLATPAK_ID}.desktop",
+                "install -Dm644 /app/bforartists/bforartists.svg /app/share/icons/hicolor/scalable/apps/$FLATPAK_ID.svg",
+                "install -Dm644 /app/bforartists/bforartists-symbolic.svg /app/share/icons/hicolor/symbolic/apps/$FLATPAK_ID-symbolic.svg",
+                "cd /app/bforartists/*.*/datafiles && mv locale /app/share/locale && ln -sf /app/share/locale locale",
+                "install -Dm644 $FLATPAK_ID.appdata.xml /app/share/metainfo/$FLATPAK_ID.metainfo.xml"
+            ],
+            "post-install": [
+                "for s in {16,22,24,32,48,64,72,96,128,192,256,512}; do rsvg-convert \"/app/bforartists/bforartists.svg\" -w \"${s}\" -h \"${s}\" -a -f png -o \"$FLATPAK_ID.png\"; [[ ! -f \"/app/share/icons/hicolor/${s}x${s}/apps/$FLATPAK_ID.png\" ]] || continue; install -p -D -m 0644 \"$FLATPAK_ID.png\" -t \"/app/share/icons/hicolor/${s}x${s}/apps/\"; done;"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/Bforartists/Bforartists/releases/download/v3.2.2/Bforartists-3.2.2-Linux.tar.xz",
+                    "sha256": "7cd3ab6099d2194e9c27bed0483fc5de8e53b8721bab76dcecb5a15ddaf223e4",
+                    "strip-components": 0
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "mv Bforartists*Linux bforartists"
+                    ]
+                },
+                {
+                    "type": "file",
+                    "path": "de.bforartists.Bforartists.appdata.xml"
```

Comment:
Why no using this: https://github.com/Bforartists/Bforartists/blob/master/release/freedesktop/org.blender.Blender.appdata.xml ? (I know it has the wrong name, but it should have been fixed upstream)

Reply:
I didn't even know that file existed. It looks like the releases are behind, but that does seem like the better option. I'll work on moving over to that file tomorrow.

Reply:
Looking through this again, I don't think it would be good to use that file. It looks like the Bforartists team just does a find/replace for names and some URLs in the Blender appdata file. The descriptions, releases, and screenshots are for Blender, which does not represent what Bforartists is.

I would prefer to use the upstream's appdata file, but I think it would be best to use a custom one until they resolve those issues.